docs/source/extensions/logging.rst

```diff
@@ -14,16 +14,36 @@
 Logging
 #######
 
-Lightning supports the most popular logging frameworks (TensorBoard, Comet, etc...).
+Supported Loggers
+=================
+
+The following are loggers we support:
+
+.. currentmodule:: pytorch_lightning.loggers
+
+.. autosummary::
+    :toctree: generated
+    :nosignatures:
+    :template: classtemplate.rst
+
+    CometLogger
+    CSVLogger
+    MLFlowLogger
+    NeptuneLogger
+    TensorBoardLogger
+    TestTubeLogger
```

Comment:
Should we avoid mentioning `TestTubeLogger`? It's deprecated and unmaintained.

Reply:
yeah, and it will be removed within v1.7? docs will be updated too in the removal PR.

Reply:
I am personally not a fan of having deprecated features in our docs at all. As an user, I'd rather not learn about this only to later notice it will be removed.

But if you prefer leaving this here until it's finally removed, that's okay. you've got my approval :P

Reply:
I agree with @carmocca, we should remove it.

Reply:
yeah makes sense.. will update :)